balancer/mist/mist_balancer.go

```diff
@@ -263,7 +264,7 @@ func (b *MistBalancer) execBalancer(ctx context.Context, balancerArgs []string)
 }
 
 func (b *MistBalancer) queryMistForClosestNode(ctx context.Context, playbackID, lat, lon, prefix string) (string, error) {
-	streamName := fmt.Sprintf("%s+%s", prefix, playbackID)
+	streamName := fmt.Sprintf("%s+%s?tag_adjust={\"%s\":1000}", prefix, playbackID, b.config.OwnRegion)
```

Comment:
The server is tagged in the [configuration](https://github.com/livepeer/livepeer-infra/pull/2042/files)